nmostesting/Config.py

```diff
@@ -190,6 +193,14 @@
 # Bash shell to use for running testssl.sh
 TEST_SSL_BASH = "bash"
 
+# IS11_ReferenceSender to test IS-11 Receivers and an IS11_ReferenceReceiver to test IS-11
+# Senders under live streaming conditions. The URL of a device API is required to access the
+# IS-04 Node (http://{device}/x-nmos/node/v1.2), IS-05 Connection (http://{device}/x-nmos/connection/v1.1)
+# and IS-11 Stream Compatibility (http://{device}/x-nmos/streamcompatibility/v1.0)
+
+IS11_REFERENCE_SENDER = ""
+IS11_REFERENCE_RECEIVER = ""
```

Comment:
This is great idea, how about I open an issue for the addition of a Mock Sender and Receiver ?

Reply:
Cannot use a mock sender because we are testing effective streaming by observing the Receiver /receivers/{receiverId}/status which not only depend on the transport parameters and SDP transport file but also effective stream content. We could have IS11_REFERENCE_SENDER_NODE_API, IS11_REFERENCE_SENDER_CONNECITON_API, IS11_REFERENCE_SENDER_COMPATIBILITY_API and the same for the reference Receiver to comply with the rule of totally independent APIs.

Reply:
We will have IS11_REFERENCE_SENDER_NODE_API, IS11_REFERENCE_SENDER_NODE_API_PORT, IS11_REFERENCE_SENDER_CONNECITON_API, IS11_REFERENCE_SENDER_CONNECITON_API_PORT to offer the full flexibility. At this time we do not need IS11_REFERENCE_SENDER_COMPATIBILITY_API